frontend/packages/topology/src/components/quick-search/utils/quick-search-utils.tsx

```diff
@@ -3,15 +3,37 @@ import { useTranslation } from 'react-i18next';
 import { TextList, TextListItem } from '@patternfly/react-core';
 import { history, removeQueryArgument } from '@console/internal/components/utils';
 import { CatalogItem } from '@console/dynamic-plugin-sdk';
-import { keywordCompare } from '@console/dev-console/src/components/catalog/utils/catalog-utils';
 import { QuickStart } from '@console/app/src/components/quick-starts/utils/quick-start-types';
 import {
   QuickStartContext,
   QuickStartContextValues,
 } from '@console/app/src/components/quick-starts/utils/quick-start-context';
 
+const keywordListCompare = (keywords: string[], item: CatalogItem): boolean => {
```

Comment:
I suggest we create a helper so we can reuse keyword logic in other places we don't want to use `fuzzy` for matching. We had similar code in 3.x with some optimizations like checking the longest keyword first (most likely not to match) to improve performance when filtering large lists of things. See

https://github.com/openshift/origin-web-common/blob/master/src/services/keyword.js

Reply:
Moved the keyword logic to a helper function.